foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,42 +1,46 @@
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 Don't forget to commit early & often! You can [reference the Commit Message lesson here](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/commit-messages)!
 
-1.  Start a new Git repo for your project.
-2.  Create a blank HTML document with a script tag (Hint: it is best practice to link an external .js file).  This game is going to be played completely from the console, so don't worry about putting anything else in there.
-3.  Your game is going to play against the computer, so begin with a function called `getComputerChoice` that will randomly return either 'Rock', 'Paper' or 'Scissors'.  We'll use this function in the game to make the computer's play. *Tip: use the console to make sure this is returning the expected output before moving to the next step!*
-4.  Write a function that plays a single round of Rock Paper Scissors.  The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner of the round like so: `"You Lose! Paper beats Rock"`
-    *   Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
-
-5.  **Important note:** you want to `return` the results of this function call, _not_ `console.log()` them. You're going to use what you `return` later on, so let's test this function by using console.log to see the results:
-
-    ~~~javascript
-    function playRound(playerSelection, computerSelection) {
-      // your code here!
-    }
+1. Start a new Git repository for your project.
+2. Create a blank HTML document with a script tag.
+    - **Tip**: We highly recommend writing the code in an external JavaScript file and linking it to the HTML file. This game is played entirely from the console, so don't worry about writing anything else in the HTML file.
+3. Your game is going to play against the computer, so begin writing a new function called `getComputerChoice` that will randomly return either `"Rock"`, `"Paper"`, or `"Scissors"`. We'll use this function to make the computer play.
+    - **Tip**: Use the console to make sure this returns the expected output before moving to the next step!
+4. Write a new function called `playRound` that plays a single round of Rock Paper Scissors. This `playRound` function should have two parameters: `playerSelection` and `computerSelection`, both taking string arguments. Then, return a string that declares the winner of the round, like so: `"You Lose! Paper beats Rock"`
+    *   Make sure your `playerSelection` parameter is case-insensitive so that users can input `rock`, `ROCK`, `RocK`, or any other variation.
+   -  **Important note:** you want to `return` the results of `playRound` call, *not* `console.log` them. You're going to use what you `return` later on, so let's test this function by using console.log to see the results:
```

Comment:
```suggestion
5. Let's test this function by using `console.log` to see the results
   - **Important note:** later on, you want to `return` the results of the `playRound` function, *not* `console.log` them. However, for now you will be using `console.log` to test the results of the function.
```
- keep the numbering but add a note since this is a step in the assignment still where the learner is expected to add `console.log`s
- Whatever we decide, let's add consistent backticks around `console.log` to be in line with the previous line 

I made some edits for this section has always felt kind of "off" to me. To me this sounds like: 
>*"We want to `return`, so let's go ahead and `console.log`"*

What do you think of these changes? They're not set in stone, just a draft change I made for you to judge.

Reply:
I think it would be better to `return` the result right at the start, never `console.log`.

We can then use `console.log` on the function call to check its output as: 

`console.log(playRound(playerSelection, computerSelection))`

Reply:
You're absolutely right! I misinterpreted that wrongly all the time and hence why it always sounded "off" to me to mention we should return but then console.log. Now it makes sense, it only took me a year!